pubsub/aws/snssqs/metadata.go

```diff
@@ -39,7 +39,7 @@ type snsSqsMetadata struct {
 	// if sqsDeadLettersQueueName is set to a value, then the messageReceiveLimit defines the number of times a message is received
 	// before it is moved to the dead-letters queue. This value must be smaller than messageRetryLimit.
 	messageReceiveLimit int64
-	// amount of time to await receipt of a message before making another request. Default: 1.
+	// amount of time to await receipt of a message before making another request. Default: 2.
```

Comment:
Changed to 2 seconds. Calls return sooner if there are messages, so polling every 1 second seemed very wasteful